### PR TITLE
Edit cluster / Azure / On Prem / k8s version upgrade

### PR DESCRIPTION
**Pull request by nLight**
## Jira Ticket
https://jira.mesosphere.com/browse/DCOS-63162

## Description of changes being made

- Azure Cloud Provider
- On Premise Provider
- Edit a Cluster
- k8s version upgrade

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md [hunk 1]
```diff
@@ -156,6 +156,69 @@ Figure 7 - Cloud Provider Form with values
...
+### Configuring an Azure cloud provider
+
+When creating an Azure cloud provider, you need an [ansible inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html) and ssh keys.
+
+#### Create a new set of credentils via CLI commands
+
+```
+az login
+```
+
```
Comment (armandgrillet) [inline pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: I would replace by the description [in my PR](https://github.com/mesosphere/dcos-docs-site/pull/2713/files) as it adds a few details and has been successfully tested by @tillt today:

Find the `SUBSCRIPTION_ID` of your account, named `id` in the output of this command:
```
az account show
```

Create the service principal for the provider:
```
az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/SUBSCRIPTION_ID"
```

The command will return data needed to create the secret:
```
{
  "appId": "APP_ID",
  "displayName": "name",
  "name": "http://name",
  "password": "PASSWORD",
  "tenant": "TENANT"
}
```

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md [hunk 2]
```diff
@@ -156,6 +156,69 @@ Figure 7 - Cloud Provider Form with values
 
 Once created, a Cloud Provider’s display name or credentials can be updated.
 
+### Configuring an Azure cloud provider
+
+When creating an Azure cloud provider, you need an [ansible inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html) and ssh keys.
+
+#### Create a new set of credentils via CLI commands
```
Comment (armandgrillet) [inline pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: s/credentils/credentials

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md [hunk 3]
```diff
@@ -156,6 +156,69 @@ Figure 7 - Cloud Provider Form with values
 
 Once created, a Cloud Provider’s display name or credentials can be updated.
 
+### Configuring an Azure cloud provider
+
+When creating an Azure cloud provider, you need an [ansible inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html) and ssh keys.
```
Comment (armandgrillet) [inline pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: s/ansible/Ansible

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ excerpt: Managing cloud providers used by Kommander
 
 Cloud providers like AWS, Azure and Google can provide the infrastructure for your Konvoy clusters. To automate their provisioning, Kommander needs authentication keys to your preferred cloud provider. It is possible to have many accounts for a single cloud provider.
 
-In order to provision new clusters and subsequently manage them, Kommander needs cloud provider credentials. Currently only AWS is supported, while Azure and GKE are coming soon.
+In order to provision new clusters and subsequently manage them, Kommander needs cloud provider credentials. Currently AWS, Azure and On Premise are supported, GKE is coming soon.
```
Comment (armandgrillet) [inline pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: s/AWS, Azure and On Premise/AWS, Azure, and On Premise

Comment (armandgrillet) [reply pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: s/supported, GKE/supported. GKE

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md [hunk 5]
```diff
@@ -156,6 +156,69 @@ Figure 7 - Cloud Provider Form with values
...
+    "displayName": "azure-cli-...",
+    "name": "http://azure-cli-...",
+    "password": "<PASSWORD>",
+    "tenant": "<TENANT>"
+}
+```
+
+#### Fill out the rest of the form
+
+- Fill out a display name for your cloud provider that you can reference later.
```
Comment (armandgrillet) [inline pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: The usage of periods at the end of bullet points is not regular. To `.` or not to `.`, that is the question 🤔 .

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md (hunk 1 above)
Comment (nLight) [reply pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: @armandgrillet feel free contributing to the PR :)

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md (hunk 5 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: I'll agree with that. A rule of thumb is if you have a group of bullets, they should be consistent. Either have periods or not. Usually a complete sentence warrants a bullet.

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md (hunk 1 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: Armand, are you going to add your description?

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md (hunk 5 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: I added bullets.

File: pages/ksphere/kommander/latest/operations/cloud-providers/index.md (hunk 1 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/latest/operations/cloud-providers/index.md]: I added Armand's description

**Pull request merged**